src/less/avatar/avatar.less

```diff
@@ -0,0 +1,111 @@
+@import "../variables/variables.less";
+
+.avatar {
+    align-items: center;
+    background-color: var(--color-marketing-teal-background-5);
+    border-radius: 48px;
+    color: var(--color-marketing-teal-foreground-5);
+    display: inline-flex;
+    font-size: @font-size-large-2;
+    font-weight: @font-weight-bold;
+    height: 48px;
+    justify-content: center;
+    overflow: hidden;
+    text-align: center;
+    width: 48px;
+}
+.avatar > svg {
+    height: 100%;
+    width: 100%;
+}
+
+.avatar > img {
+    height: 48px;
+    object-fit: cover;
+    width: 48px;
+}
+
+.avatar--teal {
+    background-color: var(--color-marketing-teal-background-5);
+    color: var(--color-marketing-teal-foreground-5);
+}
+
+.avatar--light-teal {
```

Comment:
Will this naming convention hold up if new avatars are added? For example, if an entry is added for `--color-marketing-orange-background-3` would the existing `.avatar-orange` have to be renamed `avatar--light-orange`?

Reply:
I believe these are mostly in  the center. The colors we have currently are the base. If we add new ones we will call those lighter/darker. I also think if we need to, we can have a refactor to change those colors.
We can go with the `teal3`, but then it seems arbitrary.